_source/_posts/2020-11-24-spring-boot-okta.adoc

```diff
@@ -47,7 +51,7 @@ To create a secure Spring Boot app with Okta, run `okta start spring-boot`. You'
 
 TIP: If you already have an Okta account, you can run `okta login` first.
 
-This will download our https://github.com/okta-samples/okta-spring-boot-sample[Okta Spring Boot sample], register your app on Okta, and configure it by adding your Okta settings to `src/main/resources/application.properties`.
+This will download our https://github.com/okta-samples/okta-spring-boot-sample[Okta Spring Boot sample], register your app on Okta, and configure it by adding your Okta settings to a `.okta.env` file.
```

Comment:
should we mention that this file should be at the root folder or do you think people will infer that?

Reply:
@deepu105 mentioned this in another review, but should we add some canned text here about adding this file to `.gitignore`.